pkg/daemons/control/server.go

```diff
@@ -447,6 +448,7 @@ func getNodePass(config *config.Control, serverPass string) string {
 
 func genUsers(config *config.Control, runtime *config.ControlRuntime) error {
 	passwd, err := passwd.Read(runtime.PasswdFile)
```

Comment:
variable name shadows the imported package name, but I guess that's existing logic.